helm/aphw-ddi-portal/values.yaml

```diff
@@ -38,6 +38,9 @@ container:
   messageQueueHost: namespace.servicebus.windows.net
   certificateRequestQueue: aphw-ddi-certificate-request
   jwtPrivateKey: not-a-real-private-key
+  redisHostname: dummy
+  redisPort: 6379
```

Comment:
The value in appConfig is 6380. Was 6379 intentional?

Reply:
6379 is the default for redis. App config if different overrides this.